packages/ihe-gateway-sdk/src/models/xcpd.ts

```diff
@@ -0,0 +1,41 @@
+import { Patient } from "@medplum/fhirtypes";
+import { NPIStringArray, oidStringSchema, SamlAttributes, baseResponseSchema } from "./shared";
+import { z } from "zod";
+
+export const xcpdGatewaysSchema = z.array(
+  z.object({
+    oid: oidStringSchema,
+    url: z.string(),
+    id: z.string(),
+  })
+);
+
+export type XCPDGateways = z.infer<typeof xcpdGatewaysSchema>;
+
+export type XCPDRequest = {
+  id: string;
+  cxId: string;
+  timestamp: string;
+  xcpdGateways: XCPDGateways;
+  principalCareProviderNPIs?: NPIStringArray;
+  samlAttributes: SamlAttributes;
+  patientResource: Patient;
+};
```

Comment:
There reason this isnt a zod schema is because then id have to create a schema for the FHIR Patient. Is this fine or should i do some sort of extending to append to the zod schema?